tests/robot-tests/tests/libs/common.robot

```diff
@@ -411,8 +411,8 @@ user checks page does not contain
     page should not contain    ${text}
 
 user checks page contains element
-    [Arguments]    ${element}
-    page should contain element    ${element}
+    [Arguments]    ${element}    ${wait}
+    page should contain element    ${element}    ${wait}=${timeout}
```

Comment:
`page should contain element` doesn't take a timeout argument. See https://robotframework.org/SeleniumLibrary/SeleniumLibrary.html#Page%20Should%20Contain%20Element

The seleniumlibrary keyword you need to wait is `wait until page contains element`: https://robotframework.org/SeleniumLibrary/SeleniumLibrary.html#Wait%20Until%20Page%20Contains%20Element

We already have a keyword to wrap this in common.robot: `user waits until page contains element` So I'd remove the wait from `user checks page contains element`.